diff --git a/autocomplete/AutoComplete.vue b/autocomplete/AutoComplete.vue
index 0b6a512bff5bdfb1424757da03f5ce66e65461ae..e5b7cdcf87cf0bbf296136f4ef1d2deefaaf3b03 100644
--- a/autocomplete/AutoComplete.vue
+++ b/autocomplete/AutoComplete.vue
@@ -387,10 +387,13 @@ export default {
 
                 case 'Enter':
                 case 'NumpadEnter':
-                case 'Space':
                     this.onEnterKey(event);
                     break;
 
+                case 'Space':
+                    this.onSpaceKey(event);
+                    break;
+
                 case 'Escape':
                     this.onEscapeKey(event);
                     break;
@@ -705,6 +708,11 @@ export default {
 
             event.preventDefault();
         },
+        onSpaceKey(event) {
+            if (this.focusedOptionIndex !== -1) {
+                this.onEnterKey(event);
+            }
+        },
         onEscapeKey(event) {
             this.overlayVisible && this.hide(true);
             event.preventDefault();
